hack/app_sre_pr_check.sh

```diff
@@ -11,6 +11,9 @@ if [ "$?" != "0" ]; then
     exit 1
 fi
 
+# Check PR Commit message for being conformant
+docker run --volume $(pwd):/app --rm gtramontina/commitlint:8.3.5 --from=HEAD~1
```

Comment:
I'm not sure if I'd like to see the build script fail on a commit message format, having to look up the reason for the failure on Jenkins.
If we want to have something like this, I'd recommend adding it as commit hook to catch it earlier and easier to identify. But this will make it harder to do quick and dirty commits in local branches (need to add `--no-verify`).

Also, this will only check the last commit message, not the all of a PR's commits if I'm right?

Reply:
@NautiluX Yes first of all, we are still not sure adding that into the PR's context. Wondering your thoughts... I recommend instead of adding local hooks to git, install commitlint to your local and execute it against your commit message. That can be added to the contributing docs but folks can easily find the problem by following the pipeline logs. 
Eventually, they will be redirected to https://github.com/conventional-changelog/commitlint/#what-is-commitlint (which contains the only one rule that needs to be followed for now).
The format is generic as well: https://www.conventionalcommits.org/en/v1.0.0/
The thing we need to do is tagging the commit with a suitable item.

For question two, considering we're squashing the commits into a single one before the merge, it should be enough for the pr. However, we can still apply the rules for all commit

Reply:
I'm pretty sure many PRs will include more than 1 commit, so yes we should cover all the commits